### DISCUSSION TITLE
Can individuals in my organization use Co-Pilot that the organization has paid for?

### DISCUSSION THREAD

**Discussion by dohyeondol1** [community/community#120169] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

hello. If an organization pays for and uses Co-Pilot, are individuals within that organization also free to use Co-Pilot?

**Comment by ebndev** [comment, marked as answer]
Hi @dohyeondol1!

Yes, organization owners can administer access to teams and individuals. 

You can learn more information on [Managing access for Copilot in your organization
](https://docs.github.com/en/copilot/managing-github-copilot-in-your-organization/managing-access-for-copilot-in-your-organization). For account or billing questions, you or the organization can visit our [Support page](https://support.github.com/contact).

Thank you!

P.S - I've also moved this discussion to our Copilot category for you!

**Reply by dohyeondol1** [reply]
Thank you for your quick and kind response!
